hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/ratis/TestOzoneManagerDoubleBufferWithOMResponse.java

```diff
@@ -469,7 +469,7 @@ private OMBucketCreateResponse createBucket(String volumeName,
 
     OzoneManagerProtocolProtos.OMRequest omRequest =
         OMRequestTestUtils.createBucketRequest(bucketName, volumeName, false,
-            OzoneManagerProtocolProtos.StorageTypeProto.DISK);
+            OzoneManagerProtocolProtos.StorageTypeProto.DISK, 0);
```

Comment:
Is it necessary to change the TestUtils `createBucketRequest `interface to include the client version?  Or can we just set it within the tests when needed with the static variable `ClientVersion.CURRENT_VERSION`?
If we leave the interface changed to include the clientVersion (line 535), change the call in line 472 from 
`OzoneManagerProtocolProtos.StorageTypeProto.DISK, 0`) to
`OzoneManagerProtocolProtos.StorageTypeProto.DISK, ClientVersion.CURRENT_VERSION`)

Reply:
@neils-dev `ClientVersion` has to be part of the proto OmRequest builder. We can't change it later. If we don't set it, it initializes to default which is 0. All these tests were written for client version 0. I think it's best to not change the client version for the existing tests. Most of these tests are counting on getting a LEGACY bucket layout due to the client version(`OMBucketCreateRequest.getDefaultBucketLayout()`). If we change that, the tests will still get LEGACY layout because it's currently the default for ozone. If at some point the default layout is not LEGACY then these tests will end up broken. Instead of hardcoding 0, we can replace it with `ClientVersion.DEFAULT_VERSION.toProtoValue()`